### DISCUSSION TITLE
ow to enable menu bar icon?

### DISCUSSION THREAD

**Discussion by Doh999** [mhaeuser/Battery-Toolkit#5] [Q&A]
Just discovered your app. THANK YOU for making it.
One question: How do I enable the "Battery Toolkit Menu Bar Extra" (I'm using an apple silicon Mac). I can't see an option in the settings to enable it?
Thank you

**Comment by mhaeuser** [comment]
Hey - it’s not optional and always enabled. Any chance there are too many icons already? macOS just silently hides the rest in this case, with no control over it by the apps. There’s an app called Bartender, maybe among others, to help deal with this.

**Reply by Doh999** [reply]
Thank you for replying. That is indeed the issue. I'm aware of the notch issue but it looked as if there was space for more icons. Anyway, thank you for your help. Bad UX from apple re the notch. I'm puzzled why they haven't come up with a "..." to expand menu bar or whatever.

Thanks again for your hard work on this useful app.

Best wishes, Stuart
